scalding-repl/src/main/scala/com/twitter/scalding/ScaldingILoop.scala

```diff
@@ -53,13 +53,6 @@ class ScaldingILoop
    */
   private val scaldingCommands: List[LoopCommand] = List()
 
-  /**
-   * Change the shell prompt to read scalding&gt;
-   *
-   * @return a prompt string to use for this REPL.
-   */
-  override def prompt: String = Console.BLUE + "\nscalding> " + Console.RESET
```

Comment:
I undid the change with the jline dependency. It belongs to a different patch I tried for intercepting "Ctrl-C" to stop the last submitted scalding flow without exiting the REPL.

This change was originally in connection to the first version where I was changing the prompt. I discarded the idea. I kept this particular change because method prompt is not really necessary since you can call setPrompt on base.